test_pull_requests

```diff
@@ -662,16 +666,16 @@ module Hub
         base_commit = ''
       else
         (1..num_tries).each do |i|
-          comment_pattern = /Running +\(#{ENV['BUILD_URL']}\)/
+          comment_pattern = /#{STATE_TO_CONTENT[:running]} +\(#{ENV['BUILD_URL']}\)/
```

Comment:
If you are going to do this, create a constant for Running and put that constant in the map.